docs/crictl.md

```diff
@@ -75,6 +75,16 @@ The endpoint can be set in three ways:
 - By setting environment variables `CONTAINER_RUNTIME_ENDPOINT` and `IMAGE_SERVICE_ENDPOINT`
 - By setting the endpoint in the config file `--config=/etc/crictl.yaml`
 
+If the endpoint is not set then it works as follows :
+
+- Runtime endpoint uses in order the first successful one of the following defaults:
+  - dockershim
+  - containerd
+  - cri-o
+- Image endpoint uses the runtime endpoint setting
+
+> Note: The default endpoints are now deprecated and the runtime endpoint should be set instead.
```

Comment:
mention the performance issue with trying and failing

Reply:
Sure, added.